arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+          "format": "date-time",
+          "description": "Gets or sets the occurrence time of event"
+        },
+        "submissionTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the event submission time.This value should not be confused eventTimestamp. As there might be a delay between the occurence time of the event, and the time that the event is submitted to the Azure logging infrastructure."
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "Gets or sets the Azure subscription Id"
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "Gets or sets the Azure tenant Id"
+        }
+      },
+      "description": "The Azure event log entries are of type EventData"
+    },
+    "EventDataCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EventData"
+          },
+          "description": "Gets or sets this list that includes the Azure audit logs."
+        }
+      },
+      "description": "Represents collection of events."
+    },
+    "BlobInfo": {
+      "properties": {
+        "blobUri": {
+          "type": "string",
+          "description": "Gets or sets the blob info."
+        },
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the start time of the range for this blob."
+        },
+        "endTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the end time of the range for this blob."
+        },
+        "sasToken": {
+          "type": "string",
+          "description": "Gets or sets the SAS Key for accessing Blob Storage."
+        }
+      },
+      "description": "Contains the information for blobs. Each one is used for a specific time range."
+    },
+    "BlobLocation": {
+      "properties": {
+        "blobEndpoint": {
+          "type": "string",
+          "description": "Gets or sets the blob endpoint."
+        },
+        "blobInfo": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BlobInfo"
+          },
+          "description": "Gets or sets the blob info."
+        }
+      },
+      "description": "Details about the set of blobs containing data split by time."
+    },
+    "LogDefinition": {
+      "properties": {
+        "category": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the category of the log."
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "Gets or sets the resource identifier of the resource that has emitted the log."
+        },
+        "retention": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the retention."
```

Comment:
What does retention mean? Is it the number of days or something else? Please document this accurately in the description